cfme/tests/cloud/test_keypairs.py

```diff
@@ -217,3 +217,22 @@ def test_keypair_visibility_in_tenants(appliance, child_tenant_admin_user):
 
     with child_tenant_admin_user:
         assert key_pair_obj.exists
+
+
```

Comment:
**Required** Two parts:
1) Can you add the BZ to the doc block
2) Can you mark this test with the pytest meta marker `@pytest.mark.meta(automates=[BZ_Num])`

Reply:
Okay, both should be done.